recipes-dotnet/corehost/coreclr_2.0-beta.bb

```diff
@@ -54,7 +53,18 @@ do_install() {
 	install -m 0755 ${src}/mscorlib.dll ${target}
 	install -m 0755 ${src}/System.Private.CoreLib.dll ${target}
 	install -m 0755 ${src}/SOS.NETCore.dll ${target}
+
+	# Create dev package
+	install -d ${D}/opt/dotnet-dev/
+	for i in `ls ${src}/.nuget/pkg/*.nupkg`
+	do
+		install -m 644 ${i} ${D}/opt/dotnet-dev/
```

Comment:
Right, good idea.
It does hold the corefx packages also.

I thought about changing the build number to something very high that can't be found on the internet to avoid mismatches, I'll try that quickly...